cvat/apps/documentation/faq.md

```diff
@@ -0,0 +1,72 @@
+# Frequently asked questions
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[final-newline] Missing newline character at end of file](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5666486)